src/main/java/school/lemon/changerequest/java/jdbc/Demo.java

```diff
@@ -0,0 +1,33 @@
+package school.lemon.changerequest.java.jdbc;
+
+import org.h2.jdbcx.JdbcDataSource;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static school.lemon.changerequest.java.jdbc.DbConfigHolder.*;
+
+public class Demo {
+    public static void main(String[] args) throws SQLException {
+        JdbcDataSource h2Ds = new JdbcDataSource();
+        h2Ds.setUrl(getUrl());
+
+        Connection conn = h2Ds.getConnection(getUserName(), getPassword());
+        Statement statement = conn.createStatement();
```

Comment:
Try to avoid using connections, statements with explicit resource closing.

If there is no need for such declarations - use `try-with-resources` instead:
```java
try(Connection conn = h2Ds.getConnection(getUserName(), getPassword());
     Statement statement = conn.createStatement();){
// some code
}
```